asserts/database.go

```diff
@@ -95,6 +107,41 @@ func (db *Database) atomicWriteEntry(data []byte, secret bool, subpath ...string
 	return helpers.AtomicWriteFile(fpath, data, os.FileMode(fperm), 0)
 }
 
+func (db *Database) removeEntry(subpath ...string) error {
+	fpath := filepath.Join(db.root, filepath.Join(subpath...))
+	return os.Remove(fpath)
+}
+
+func (db *Database) symlinkEntry(entry string, subpath ...string) error {
+	fpath := filepath.Join(db.root, filepath.Join(subpath...))
+	// TODO: move to helpers/helpers.go
```

Comment:
I actually hope we can move the content in helpers to packages slightly more indicative of their function in the system. Names such as utils, helpers, tools, etc, tend to creep up for lack of clear focus.